internal/controllers/common_controller.go

```diff
@@ -152,18 +152,6 @@ func setStatusUnpackPending(ext *ocv1alpha1.ClusterExtension, message string) {
 	})
 }
 
-// TODO: verify if we need to update the installBundle status or leave it as is.
-func setStatusUnpacking(ext *ocv1alpha1.ClusterExtension, message string) {
-	ext.Status.InstalledBundle = nil
-	apimeta.SetStatusCondition(&ext.Status.Conditions, metav1.Condition{
-		Type:               ocv1alpha1.TypeUnpacked,
-		Status:             metav1.ConditionFalse,
-		Reason:             ocv1alpha1.ReasonUnpacking,
-		Message:            message,
-		ObservedGeneration: ext.GetGeneration(),
-	})
-}
-
 func setStatusUnpacked(ext *ocv1alpha1.ClusterExtension, message string) {
```

Comment:
I think the others, `setStatusUnpack[Pending|Failed]` can go away too

Reply:
Technically yes, `unpackPending` can go away (we still need failed when it returns an error). But there is a caveat - which I have explained in detail in here: https://github.com/operator-framework/operator-controller/issues/929. It would break our tests, and we would have to continue implementing mock loader and storer. I wanted to keep that as a separate task as it needs considerable effort and not comprise on the tests we have currently - which is why have removed the `unpacking` state which is not being used anywhere.

Reply:
yep seems fine for a followup on `unpackPending` 👍